forgerock-auth/src/main/java/org/forgerock/android/auth/ConfigHelper.kt

```diff
@@ -103,6 +97,13 @@ internal class ConfigHelper: ConfigInterface {
             return false
         }
 
+        @JvmStatic
+        fun closeSession(context: Context, cachedOptions: FROptions?) {
+            val config = Config()
+            val previousOptions = cachedOptions ?: loadFromPreference(context)
+            config.init(context, previousOptions)
+            config.sessionManager.close()
+        }
```

Comment:
@witrisna  This is a another approach i can think of. making config package private and invoking directly